### PR TITLE
Move multi touch action test to Actions API

### PR DESCRIPTION
**Pull request by shridharkalagi**

### CODE REVIEW

File: src/test/java/com/appium/gesture/GestureTest.java [hunk 1]
```diff
@@ -166,22 +166,35 @@ private void verticalSwipe(String locator) throws InterruptedException {
...
-                        .moveTo(ElementOption.element(slider, sizeSlider.width / 2, sizeSlider.height / 2));
-        TouchAction touchAction2 =
-                new TouchAction(driver).press(ElementOption.element(slider1, 0, sizeSlider1.height / 2)).waitAction(WaitOptions.waitOptions(Duration.ofSeconds(1)))
-                        .moveTo(ElementOption.element(slider1, sizeSlider1.width / 2, sizeSlider1.height / 2));
-        new MultiTouchAction(driver).add(touchAction1).add(touchAction2).perform();
-        Thread.sleep(2000);
+        Point source1 = slider.getLocation();
+        Point source2 = slider1.getLocation();
+
+        PointerInput pointerInput1 = new PointerInput(PointerInput.Kind.TOUCH, "finger1");
```
Comment (SrinivasanTarget) [inline src/test/java/com/appium/gesture/GestureTest.java]: Are you sure `PointerInput` accepts it as `finger1`?

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

File: src/test/java/com/appium/gesture/GestureTest.java (hunk 1 above)
Comment (shridharkalagi) [reply src/test/java/com/appium/gesture/GestureTest.java]: Yes it works

**Comment by saikrishna321** [comment]
@shridharkalagi does this work on iOS ?

File: src/test/java/com/appium/gesture/GestureTest.java (hunk 1 above)
Comment (saikrishna321) [reply src/test/java/com/appium/gesture/GestureTest.java]: @SrinivasanTarget Selenium handles it 

    public PointerInput(PointerInput.Kind kind, String name) {
        this.kind = (PointerInput.Kind)Objects.requireNonNull(kind, "Must set kind of pointer device");
        this.name = (String)Optional.ofNullable(name).orElse(UUID.randomUUID().toString());
    }

**Review by saikrishna321 — APPROVED** [review]

**Pull request closed without merging**